attributes/default.rb

```diff
@@ -1,3 +1,5 @@
 default['osl-php']['packages'] = []
 default['osl-php']['php_packages'] = []
 default['osl-php']['use_ius'] = false
+# List PHP versions that require IUS archive repo: https://ius.io/LifeCycle/#php
+node.default['osl-php']['ius_archive_versions'] = %w(5.6 7.0)
```

Comment:
This should just be ``default['blah']...``. You don't use ``node.default`` in attribute files.